tools/gradle-plugin/src/test/kotlin/io/github/composegears/valkyrie/gradle/ValkyrieGradlePluginTest.kt

```diff
@@ -0,0 +1,349 @@
+                    id("com.android.library")
+                    id("io.github.composegears.valkyrie")
+                }
+
+                android {
+                    namespace = "a.b.c"
+                    compileSdk = 33
+                }
+            """.trimIndent(),
+        )
+        root.writeTestSvgs(sourceSet = "main")
+
+        // when
+        val result = runTask(root, TASK_NAME).build()
+
+        // then
+        assertThat(result.task(":$TASK_NAME")?.outcome).isEqualTo(SUCCESS)
+        assertThat(result.output).contains("Generated 4 ImageVectors in package a.b.c")
+    }
+
+    @Test
+    fun `Package name needs to be explicitly set if AGP isn't applied`() {
+        // given
+        root.resolve("build.gradle.kts").writeText(
+            """
+                plugins {
+                    kotlin("jvm")
+                    id("io.github.composegears.valkyrie")
+                }
+            """.trimIndent(),
+        )
+        root.writeTestSvgs(sourceSet = "main")
+
+        // when
+        val result = runTask(root, TASK_NAME).buildAndFail()
+
+        // then
+        assertThat(result.output).contains("Couldn't automatically estimate package name")
+    }
+
+    @Test
+    fun `Setting custom package name`() {
+        // given
+        root.resolve("build.gradle.kts").writeText(
+            """
+                plugins {
+                    kotlin("jvm")
+                    id("io.github.composegears.valkyrie")
+                }
+
+                valkyrie {
+                    packageName = "my.custom.package"
+                }
+            """.trimIndent(),
+        )
+        root.writeTestSvgs(sourceSet = "main")
+
+        // when
+        val result = runTask(root, TASK_NAME).build()
+
+        // then
+        assertThat(result.task(":$TASK_NAME")?.outcome).isEqualTo(SUCCESS)
+        assertThat(result.output).contains("Generated 4 ImageVectors in package my.custom.package")
+    }
+
+    @Test
+    fun `Generate from SVGs with default config`() {
+        // given
+        root.resolve("build.gradle.kts").writeText(
+            """
+                plugins {
+                    kotlin("jvm")
+                    id("io.github.composegears.valkyrie")
+                }
+
+                valkyrie {
+                    packageName = "x.y.z"
+                }
+            """.trimIndent(),
+        )
+        root.writeTestSvgs(sourceSet = "main")
+
+        // when
+        val result = runTask(root, TASK_NAME).build()
+
+        // then
+        assertThat(result.task(":$TASK_NAME")?.outcome).isEqualTo(SUCCESS)
+        assertThat(result.output).contains("Generated 4 ImageVectors in package x.y.z")
+        assertThat(result.output).contains("LinearGradient.kt")
+        assertThat(result.output).contains("RadialGradient.kt")
+        assertThat(result.output).contains("ClipPathGradient.kt")
+        assertThat(result.output).contains("LinearGradientWithStroke.kt")
```

Comment:
```suggestion
        assertThat(result.output).contains(
            "Generated 4 ImageVectors in package x.y.z",
            "LinearGradient.kt",
            "RadialGradient.kt",
            "ClipPathGradient.kt",
            "LinearGradientWithStroke.kt",
        )
```

`contains` supports `vararg`.